content/ops/contingency-plan.md

```diff
@@ -0,0 +1,122 @@
+* Each team member core to cloud.gov operations maintains a list of the phone numbers of the other core team members, stored offline in a protected way (such as in the contacts app in their GSA phone or in a file on their GSA laptop).
+* Thorough documentation
+* The basic steps for rebuilding cloud.gov are written in our [cg-provision repository](https://github.com/18f/cg-provision).
+* We back take snapshots of all RDS instances and store them for at least 30 days.
+* For AWS availability we use hot multi-AZ replication. By doing that we inherit AWS controls for alternate processing.
+* Telecommunications practices Appendix C-2 and C-3 (inherit a lot from AWS).
+
+---
+
+## Updating this contingency plan
+
+At least once a year the cloud.gov team will review this contingency plan and update it as needed.
+
+---
+
+## How this document works
+
+This plan is going to be most effective if all core cloud.gov team members know about it, remember that it exists, have the ongoing opportunity to give input based on their expertise, and keep it up to date. Here’s how we’ll make that happen.
+
+* Initial review and approval:
+  * cloud.gov team leads propose this draft to the team,&nbsp;inviting all team members to review and comment on it. The team leads approve an initial version to be published on docs.cloud.gov as our working document. Team leads coordinate with 18F Director of Infrastructure to make sure our plan complies with all necessary requirements.
+* Distribution:
+  * We publish this plan on [https://docs.cloud.gov/](https://docs.cloud.gov) (which is sourced from [https://github.com/18F/cg-docs](https://github.com/18F/cg-docs)) in our Operations section, alongside our Security Incident Response Guide and other key team policies. Our [Onboarding Checklist](https://github.com/18F/cg-product/blob/master/OnboardingChecklist.md)&nbsp;includes an item for all new team members to read this plan as well as other key team policies when they join.
+* How we review and update this plan and communicate those updates (to address changes to our team or systems, along with any problems we find while testing or using this plan):
+  * The cloud.gov team is responsible for maintaining this document and updating it as needed. Any change to it must be approved and peer reviewed by at least another member of the team.
+  * All changes to the plan should be communicated to the rest of the team.
+* How we protect this plan from unauthorized modification:
+  * This plan is stored in a public GitHub repository ([https://github.com/18F/cg-docs](https://github.com/18F/cg-docs)) with authorization to modify it limited to 18F staff by GitHub access controls. cloud.gov team policy is that 18F staff may not directly make changes to this document; they must propose a change by making a pull request and ask another team member to review and merge the pull request.
```

Comment:
I'd just refer to the 18F-wide policy:

https://handbook.18f.gov/github/#pull-requests

We can go into more detail there if necessary.